docs/getting-started-guides/vagrant.md

```diff
@@ -412,6 +412,24 @@ export KUBERNETES_MASTER_MEMORY=1536
 export KUBERNETES_NODE_MEMORY=2048
 ```
 
+#### I want to set proxy settings for my Kubernetes cluster boot strapping!
+
+If you are behind of a proxy, you need to install vagrant proxy plugin and set the proxy settings by 
+
+```sh
+vagrant plugin install vagrant-proxyconf
+export KUBERNETES_HTTP_PROXY=http://username:password@proxyaddr:proxyport
+export KUBERNETES_HTTP_PROXY=https://username:password@proxyaddr:proxyport
+```
+
+Optionally, you can specify addresses of no proxy for kubernetes build, for example
```

Comment:
This reads awkwardly to me... 

Optionally you can specify addresses to not proxy?